descwl_coadd/coadd.py

```diff
@@ -479,7 +484,7 @@ def make_warps(
     # stamp size for input and output psf and just zero out wherever there is
     # no data
 
-    verifys = [True, True, False, True]
+    verifys = [False]*4  # HACK [True, True, False, True]
```

Comment:
This verification is an important check.  I think it is pretty fast too.  Why turn it off?

Reply:
see discussion elsewhere, this should go later when trimming warps to the cell coadd region

Reply:
It was never about the speed. When you warp an image, the corners are inevitably going to have `NO_DATA` regions, even if we restrict to the patch's bounding box.

Reply:
why did our code work?

Reply:
It did not work on this branch with patch sized bounding boxes (and rightly so) on this branch, i.e., after separating out warping from coaddition. It might have worked previously if you were looking only at small regions corresponding to cells.

Reply:
OK I misunderstood.  Yes, the check would fail for patch sized bbox but not for cell coadd bbox